apps/consumer-client/src/pages/examples/gpc-proof.tsx

```diff
@@ -108,6 +126,26 @@ export default function Page(): JSX.Element {
           }}
         />
         <br />
+        Prescribed entries (or empty for none):
```

Comment:
This could use a but more explanation of what this field does:
    Prescribed entries to filter which PODs user can select.  These entries must be revealed in the configuration.  (Use empty for none.)